schema/queue/message/impl/RabbitMqMessageSubscriber.go

```diff
@@ -59,21 +59,22 @@ func (cs *CommonMessageSubscriber) Handler(msgDelivery amqp.Delivery) {
 	result := &p_buff.MessageGroupBatch{}
 	err := proto.Unmarshal(msgDelivery.Body, result)
 	if err != nil {
-		cs.Logger.Fatal().Err(err).Str("Method", "\"Handler\"").Msg("Can't unmarshal proto")
+		//TODO In this method I need to stop execution in case of error. can I use here Fatal or Panic ?
```

Comment:
Please, use Error logging here for now and leave a comment that we need to return error to the call side. The same applies for other places here where you use `Fatal`